### ISSUE TITLE
Overlaps between 5'UTRs in the same conditions

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#1024]
when we discussed
https://github.com/pombase/pombase-chado/issues/1019

we talked about reporting the changes in 5' UTR overlaps  in the publication (I need to check  what has been published on this).

So we would need to know the difference in 5'UTR overlap between the old and the new dataset.

(I said I would open a ticket, but I forgot)

**Comment by ValWood** [comment]
after https://github.com/pombase/pombase-chado/issues/1019

**Comment by ValWood** [comment]
https://github.com/pombase/pombase-chado/issues/1049#issuecomment-1350428878
might be useful here.

**Comment by kimrutherford** [comment]
> https://github.com/pombase/pombase-chado/issues/1049#issuecomment-1350428878
> might be useful here.

It's tricky because the old UTRs are no longer in Chado, so can't be queried with SQL.  This will probably need a bit of Perl (or Python)

**Comment by kimrutherford** [comment]
Is this something we should look at soon?

**Comment by ValWood** [comment]
NOt really. It might be intertesting to report this in the genome submission paper UTR section.
If you could do it sometime in the next 6 months  (we will submit the genome shortly, but this is probably  the last paper in the queue of 3).

**Comment by kimrutherford** [comment]
> If you could do it sometime in the next 6 months

Time's nearly up. :-)

**Comment by ValWood** [comment]
Deprioritised we  now have 2 other papers to submit before the genome update ;)

**Comment by ValWood** [comment]
We need to disambiguate the for_pub labels ;)
